ciphers/monoalphabetic_cipher.cpp

```diff
@@ -0,0 +1,50 @@
+#include <bits/stdc++.h>
+using namespace std;
+unordered_map<char,char> hashMap;
+
+string encrypt(string msg)
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.